source/_includes/archive_post.html

```diff
@@ -1,5 +1,5 @@
 {% capture category %}{{ post.categories | size }}{% endcapture %}
 <time datetime="{{ post.date | datetime | date_to_xmlschema }}" pubdate class="flex items-center">
-    {{ post.date | date: "<span class='month'>%b</span> <span class='day'>%d</span> <span class='md:hidden'>%Y</span>"}}
+    {{ post.date | date: "<span>%-d</span>&nbsp;<span>%B</span>&nbsp;<span>%Y</span>"}}
 </time>
-<h1 class="text-black"><a href="{{ root_url }}{{ post.url }}">{{post.title}}</a></h1>
+<h2 class="mb-1"><a href="{{ root_url }}{{ post.url }}">{{post.title}}</a></h2>
```

Comment:
[nit-pick] why choose h2 over h1 here? Having `h1` tags improves SEO ranking.

Reply:
These are links to posts in the list, `h1`s are for page titles.